assets/js/blocks/catalog-sorting/block.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "woocommerce/catalog-sorting",
+	"version": "1.0.0",
+	"title": "Catalog Sorting",
+	"description": "Enable customers to change the sorting order of the products.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"align": true,
+		"color": {
+			"text": true
+		},
+		"typography": {
+			"fontSize": true,
+			"__experimentalFontFamily": true
+		},
+		"inserter": false
```

Comment:
Is this ok? You shouldn't be able to insert it on its own, right?

Reply:
Thanks, I'll see if that's possible 👍

Reply:
Thanks for your investigation. 

If restricting the block to WooCommerce templates isn't possible, can we register the block for Site Editor only, which means that the block will only be available when editing templates? I believe we did the same for the Classic Template block.

Reply:
I've pushed a change to achieve this, I used the classic template as inspiration. Let me know what you think.

Reply:
@albarin I'm a bit concerned about performance with the `subscribe` function as it may run multiple times without debounce/throttle. I see we're using this technique for the Classic Template block:

https://github.com/woocommerce/woocommerce-blocks/blob/ff26c8112284d7a1c2a2f533dc7530cf8ee35802/src/BlockTypesController.php#L242-L252

I think we can do the same for this block. By adding the `CatalogSorting` to that array, we can have a smaller diff and avoid complex JS handling. What do you think?

Reply:
Thanks I didn't know about this, it's a much better solution. I have updated it on both PRs. 😄